src/fieldmap.c

```diff
@@ -63,7 +63,7 @@ const struct MapHeader * mapconnection_get_mapheader(struct MapConnection * conn
     return Overworld_GetMapHeaderByGroupAndId(connection->mapGroup, connection->mapNum);
 }
 
-void not_trainer_hill_battle_pyramid(void)
+void not_trainer_tower_battle_pyramid(void)
```

Comment:
Battle Pyramid didn't exist in FRLG.

Reply:
Correct. The name just isn't synced, it doesnt have anything to do with Trainer Tower either.